helix-term/src/ui/completion.rs

```diff
@@ -178,17 +178,12 @@ impl Completion {
                     });
 
                     // apply additional edits, mostly used to auto import unqualified types
-                    let resolved_additional_text_edits = if item.additional_text_edits.is_some() {
-                        None
-                    } else {
-                        Self::resolve_completion_item(doc, item.clone())
-                            .and_then(|item| item.additional_text_edits)
-                    };
```

Comment:
Was this change necessary? We avoided additional lookups if the item was already fully resolved.

Reply:
I agree with you, but still we need to check whether list is empty or not.